### ISSUE TITLE
feature request: soft-wrap

### ISSUE THREAD

**Issue by bluelhf** [RoyTakanen/pastebinUI#7]
a soft-wrap toggle would make it easier to read pastes with very long lines :)
implementation could be as simple as a button that toggles `white-space: pre-wrap;` on the `<pre/>` tag containing the paste contents :D

**Comment by RoyTakanen** [comment]
Yes. This is something that was implemented in the previous version. I will add some sort of toggle for that as you suggested when the refactor is done. Thanks for the feedback!
